org.tryton.Tryton-66.yml

```diff
@@ -0,0 +1,31 @@
+id: org.tryton.Tryton-66
```

Comment:
is `66` some sort of version? Usually this shouldn't be part of the app-id. Unless it's parallel instable because long term support.

Reply:
Yes it is the series see https://discourse.flathub.org/t/how-to-publish-multiple-major-version/2205

Reply:
Are 6.x releases incompatible between each other? If not, I'd recommend sticking to "Tryton-6".

Reply:
Yes they are incompatible.